### PR TITLE
Added vendor prefixes for flexbox

### LINKED ISSUE #218: Add vendor prefix for flexbox
Page layout looks broken on older versions of Safari.

img

(reported by @chadsansing originally [here](https://github.com/mozilla/teach.webmaker.org/issues/1199))

### PR DESCRIPTION
**Pull request by mmmavis**
This fixes #218.

@Pomax I'm assigning this to youuuuu

### CODE REVIEW

File: stylesheets/styles.css [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
 .wrapper {
   display: -webkit-box;
-  display: -moz-box;
```
Comment (mmmavis) [inline stylesheets/styles.css:7]: guess we don't need this anymore... unless we still want to support super outdated FF?

**Comment by Pomax** [comment]
the CSS looks sensible, but I don't have a mac to test on right now; can someone else do this review pass?

**Comment by mmmavis** [comment]
@Pomax sure, or i can show it to you when we are both at the office later today :smile:

**Comment by mmmavis** [comment]
got a :+1: from @Pomax :dancer:

**Pull request merged**